ReservacionVuelos/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.4
```

Comment:
Why is this version of RUby in your projecto?

Reply:
@YilKanda @Blueheart90 mentioned that those files came o where generated automatically.. should they be removed?

Reply:
The Ruby files was removed from project, in this commit d856a3799aab882c4db54c22d5ff1d574b47572e